src/composables/lastTransactions.ts

```diff
@@ -0,0 +1,21 @@
+import { computed } from '@vue/composition-api';
+import { ITransaction } from '../types';
+import store from '../store';
+import {
+  COUNT_OF_DASHBOARD_TRANSACTION,
+  sortTransaction,
+} from '../popup/utils';
+
+export const useLastTransaction = () => {
+  const availableTokens = computed<Record<string, ITransaction[]>>(
+    () => (store as any).state?.fungibleTokens?.transactions,
+  );
```

Comment:
you have set either wrong name of variable or have wrong getter

Reply:
Also please use useState here

Reply:
i can't, after @peronczyk  change we cant use useState, useGetter in composables